sunpy/map/mapbase.py

```diff
@@ -221,6 +221,9 @@ def __repr__(self):
            dim=u.Quantity(self.dimensions),
            scale=u.Quantity(self.scale),
            tmf=TIME_FORMAT) + self.data.__repr__())
+                                                                        
+    def _modified_copy(self, data, meta, **kwargs):
+        return GenericMap(data, meta, **kwargs)
```

Comment:
I would persist `plot_settings` here as well (i.e copy plot settings from `self` to the new map).